packages/bento-design-system/src/Field/Config.ts

```diff
@@ -17,6 +18,7 @@ export type FieldConfig = {
   tip: {
     icon: (props: IconProps) => Children;
     iconSize: IconProps["size"];
+    placement: Placement;
```

Comment:
Would something like this work to avoid having to export the type, but still have them automatically synced?

```typescript
import { Placement as _Placement } from "@floating-ui/core";

export type Placement = `${_Placement}`;
```